fix created Module

devF4B1J opened the pull request:
Created some Modules to improve the readability.

devF4B1J committed:
fix created Module

CodingSinceThe80s committed:
Merge branch 'dev' into fix-create-modules

CodingSinceThe80s commented on package.json:
```diff
@@ -9,7 +9,7 @@
   },
   "dependencies": {
-    "@material-ui/core": "4.10.0",
+    "@material-ui/core": "^4.10.0",
```
Warum "bis zu"? Gibt es eine konkrete Anforderung dahinter?

CodingSinceThe80s committed:
Update package.json

Änderung war durch eine versehentliche npm-Nutzung verursacht, wird auf 4.10. wiederhergestellt
